### DISCUSSION TITLE
Some questions for the rgb code

### DISCUSSION THREAD

**Discussion by cymqqqq** [RGB-WG/rgb#78] [Q&A]
What about the ideas of defining the fixed system-level interfaces?(Like rgb20, rgb21) What advantages or disadvantages?

**Comment by dr-orlovsky** [comment]
Not sure I understand the question. What do you mean by "fixed system-level"?

Overall being flexible is better than fixed - more people can contribute and propose new use cases which are jot covered by a fixed set, which for consensus systems sets the things in stone forever.
